src/kv/bulk.rs

```diff
@@ -0,0 +1,67 @@
+extern crate base64;
```

Comment:
what is the point of `extern crate`? that was deprecated in rust 2018

Reply:
idk! i thought i had  dropped this  but it maybe snuck back in? can update